common/SmokeTests/SmokeTest/Update-Dependencies.ps1

```diff
@@ -20,6 +23,9 @@ $PACKAGE_REFERENCE_XPATH = '//Project/ItemGroup/PackageReference'
 # Matches the dev.yyyymmdd portion of the version string
 $DEV_DATE_REGEX = 'dev\.(\d{8})'
 
+$baselineVersionDate = $null;
+
+$RELEASE_FEED_URL = 'https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json'
```

Comment:
is Release feed the right name? This feed contains both the dev/beta and released packages.